WordPress/Classes/ViewRelated/Stats/SiteStatsDashboard.storyboard

```diff
@@ -41,19 +41,29 @@
                                 </constraints>
                             </view>
                             <containerView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="J6j-x0-sMS">
-                                <rect key="frame" x="0.0" y="90" width="375" height="577"/>
+                                <rect key="frame" x="0.0" y="90" width="375" height="557"/>
                                 <connections>
                                     <segue destination="83f-eK-Wfs" kind="embed" id="Vur-EF-t51"/>
                                 </connections>
                             </containerView>
+                            <view contentMode="scaleToFill" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="zMO-yu-sCx" customClass="JetpackBannerView" customModule="WordPress" customModuleProvider="target">
+                                <rect key="frame" x="0.0" y="647" width="375" height="20"/>
```

Comment:
From the designs, the height of the banner should be 44. At runtime, it's also different (see below)
<p align=center>
<img height="450" src="https://user-images.githubusercontent.com/34376330/176975150-4f33bcf0-93bd-453d-bfb0-9bcf39d6a667.png">
</p>

Reply:
Thanks for spotting that discrepancy, will fix!

Reply:
Fixed now, it was 6px too short. Updated now to 44px.